tests/wol/test_wol.py

```diff
@@ -385,3 +385,165 @@ def validate_wol_packets(pkts):
         pkts_validator=validate_wol_packets
     ):
         duthost.shell("wol %s %s -b" % (random_dut_port, target_mac))
+
+
+@pytest.mark.parametrize("password", ["192.168.0.256", "q1:11:22:33:44:55"])
+def test_invalid_password(
+    duthost,
+    get_connected_dut_intf_to_ptf_index,
+    password
+):
+    target_mac = "1a:2b:3c:d1:e2:f7"
+    connected_dut_intf_to_ptf_index = get_connected_dut_intf_to_ptf_index
+    random_dut_port, random_ptf_port = random.choice(connected_dut_intf_to_ptf_index)
+    logging.info("Test with random dut port %s and ptf port index %s" % (random_dut_port, random_ptf_port))
+    exception_catched = False
+    try:
+        duthost.shell("wol %s %s -b -p %s" % (random_dut_port, target_mac, password))
```

Comment:
`duthost.shell()` should be able to return a dict which contains `rc` and `stderr`. Why the testcase expect Exception?

Reply:
It will raise exception before return dict, so we need to catch the exception.

Reply:
I see.